api/overdrive.py

```diff
@@ -311,6 +319,11 @@ def get_fulfillment_link(self, patron, pin, overdrive_id, format_type):
             return response
 
     def get_fulfillment_link_from_download_link(self, patron, pin, download_link):
+        # If this for Overdrive's streaming reader, and the link expires,
+        # the patron can go back to this url again to get a new one.
+        auth_url = flask.request.url
```

Comment:
I'd like to keep Flask references out of the circulation API. Could this URL be passed in as an argument?